Add `update_version` to the default fields for `plugin` and `theme` commands

sejas opened the pull request:
<!--

Thanks for submitting a pull request!

Please review our contributing guidelines if you haven't recently: https://make.wordpress.org/cli/handbook/contributing/#creating-a-pull-request

Here's an overview to our process:

1. One of the project committers will soon provide a code review: https://make.wordpress.org/cli/handbook/code-review/
2. You are expected to address the code review comments in a timely manner (if we don't hear from you in two weeks, we'll consider your pull request abandoned).
3. Please make sure to include functional tests for your changes.
4. The reviewing committer will merge your pull request as soon as it passes code review (and provided it fits within the scope of the project).

You can safely delete this comment.

-->
- Related:
  - https://github.com/wp-cli/wp-cli/issues/5859
  - https://github.com/wp-cli/extension-command/issues/198

## Description

I'm adding the field `update_version` as a default field for `plugin` and `theme` commands. If no `update_version` is available we return an empty string following the same pattern as `version` itself.

Now `wp plugin list` and `wp theme list` display a table similar to:
```
+------------+----------+--------+---------+----------------+-------------+
| name       | status   | update | version | update_version | auto_update |
+------------+----------+--------+---------+----------------+-------------+
| akismet    | inactive | none   | 5.3     |                | off         |
| hello      | inactive | none   | 1.7.2   |                | off         |
| Zombieland | inactive | none   | 0.1.0   |                | off         |
| no-mail    | must-use | none   |         |                | off         |
| polyfills  | must-use | none   |         |                | off         |
+------------+----------+--------+---------+----------------+-------------+
```

```
+-------------------+----------+--------+---------+----------------+-------------+
| name              | status   | update | version | update_version | auto_update |
+-------------------+----------+--------+---------+----------------+-------------+
| twentytwentyfour  | active   | none   | 1.0     |                | off         |
| twentytwentythree | inactive | none   | 1.3     |                | off         |
| twentytwentytwo   | inactive | none   | 1.6     |                | off         |
+-------------------+----------+--------+---------+----------------+-------------+
```



## Testing instructions

1. Run `composer behat -- features/plugin-update.feature`
2. Run `composer behat -- features/plugin.feature`
3. Run `composer behat -- features/upgradables.feature`
4. Run `composer behat -- features/theme.feature`
5. Observe the tests pass

schlessera commented on features/plugin.feature:
```diff
@@ -131,8 +131,8 @@ Feature: Manage WordPress plugins
     Then STDOUT should be a table containing rows:
-      | name               | status   | update    | version |
-      | wordpress-importer | active   | available | 0.5     |
+      | name               | status   | update    | version | update_version   |
+      | wordpress-importer | active | available | 0.5     | {UPDATE_VERSION} |
```
Alignment needs a fix here.

schlessera commented on features/theme.feature:
```diff
@@ -97,8 +101,8 @@ Feature: Manage WordPress themes
     Then STDOUT should be a table containing rows:
-      | name  | status   | update    | version   |
-      | p2    | active   | available | 1.4.1     |
+      | name  | status   | update    | version   | update_version |
+      | p2    | inactive | available | 1.4.1     | {UPDATE_VERSION} |
```
Alignment needs to be fixed here.

schlessera commented on features/theme.feature:
```diff
@@ -77,10 +77,14 @@ Feature: Manage WordPress themes
     Then STDOUT should be a table containing rows:
-      | name  | status   | update    | version   |
-      | p2    | inactive | available | 1.4.2     |
+      | name  | status   | update    | version   | update_version   |
+      | p2    | active | available | 1.4.2     | {UPDATE_VERSION} |
```
Alignment needs a fix here.

swissspidy reviewed (commented):
Besides the alignment details this looks good 👍

sejas commented:
Thanks for the quick review! I fixed the alignment issues. I also updated some comments on the functions.

And I realized that the `json` format is returning null instead of an empty string. Is that something I need to change?
I would appreciate where that part of the code lives.

` wp plugin list --format=json`

```
[
    {"name":"akismet","status":"active","update":"none","version":"5.3","update_version":null}
    ,{"name":"hello","status":"inactive","update":"none","version":"1.7.2","update_version":null}
    ,{"name":"Zombieland","status":"inactive","update":"none","version":"0.1.0","update_version":null}
    ,{"name":"no-mail","status":"must-use","update":"none","version":"","update_version":null}
    ,{"name":"polyfills","status":"must-use","update":"none","version":"","update_version":null}
]
````

The table, and csv format are working correctly returning an empty string.

I'll also solve the merging conflicts.

sejas commented:
> And I realized that the json format is returning null instead of an empty string. Is that something I need to change?
I would appreciate where that part of the code lives.

I found it:  https://github.com/wp-cli/extension-command/pull/380/commits/9749bd2a293c12609ef4df95741453438e9029d1
